src/pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index.jsx

```diff
@@ -0,0 +1,307 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Row, Col, Space, Collapse, Skeleton, Empty, Typography,
+} from 'antd';
+import _ from 'lodash';
+import { useSelector, useDispatch } from 'react-redux';
+import { Vega } from 'react-vega';
+import PropTypes from 'prop-types';
+import loadProcessingSettings from '../../../../../redux/actions/experimentSettings/processingConfig/loadProcessingSettings';
+import PlotStyling from '../../../../../components/plots/styling/PlotStyling';
+import { updatePlotConfig, loadPlotConfig } from '../../../../../redux/actions/componentConfig';
+import Header from '../../../../../components/plots/Header';
+import { generateSpec } from '../../../../../utils/plotSpecs/generateHeatmapSpec';
+import { loadGeneExpression, loadMarkerGenes, setGeneOrder } from '../../../../../redux/actions/genes';
+import { loadCellSets } from '../../../../../redux/actions/cellSets';
+import PlatformError from '../../../../../components/PlatformError';
+import Loader from '../../../../../components/Loader';
+import populateHeatmapData from '../../../../../components/plots/helpers/populateHeatmapData';
+import HeatmapControls from '../../../../../components/plots/styling/heatmap/HeatmapControls';
+
+const { Text } = Typography;
+const { Panel } = Collapse;
+
+const plotUuid = 'markerHeatmapPlotMain';
+const plotType = 'markerHeatmap';
+
+const route = {
+  path: 'marker-heatmap',
+  breadcrumbName: 'Marker-Heatmap',
+};
+const HeatmapPlot = ({ experimentId }) => {
```

Comment:
I would call this component `MarkerHeatmap`, just to distinguish it from the heatmap component easier